src/Apps/Order2/Utils/errors.ts

```diff
@@ -0,0 +1,21 @@
+export { CheckoutMutationError } from "Apps/Order/Components/ExpressCheckout/Util/mutationHandling"
+
```

Comment:
Previously imported from Apps/Order (not Order2). I moved it to an export from here to keep things neat.